### PR TITLE
Update "Your First ASP.NET 5 Web App Using Visual Studio"

### LINKED ISSUE #1076: Update 'Your First ASP.NET 5 Web App Using Visual Studio' Tutorial to Match Latest Released
Currently as of the writing of this issue with Visual Studio 2015 Update 1 and the latest ASP.NET and tools installed RC1 Update 1 etc the current ASP.NET 5 Web App for Visual Studio is way out of date.

Everything is find and working in the tutorial until you get to the "Review the project" section and then the project layout is drastically different. Then if you take a closer look the screenshot of the running app looks different too. The current default template creates one with registration/login that is using EF7 already with a Migrations folder and is using ViewModels and Services.

Seems like the tutorial now needs to address these things and also do it in a way using the existing EF7 context and models. As the current default already did some of the work the tutorial is trying to tell the reader to do, so it gets confusing to follow making it rather useless at this point starting at the "Review the Project" section.

I bring this issue up as when using a tutorial you want the reader to have a friction-less experience as you are attempting to give them a positive feeling about what they are learning. Right now the current 'Your First ASP.NET 5 Web App Using Visual Studio' tutorial does not provide a friction-less experience.

### PR DESCRIPTION
**Pull request by danroth27**
Fixes #1076 

Update screen shots and did some minor text update. The tutorial should now match the latest released bits.

@Rick-Anderson @rachelappel @shirhatti @CreepyGnome Please review

### CODE REVIEW

File: aspnet/tutorials/your-first-aspnet-application.rst [hunk 1]
```diff
@@ -75,10 +75,12 @@ Visual Studio creates some initial folders and files for your project. The prima
 
    * - HomeController.cs     
      - This ``controller`` contains the classes that handle incoming browser requests, retrieve model data, and then specify view templates that return a response to the browser. 
-	 
+
+In addition to these files the project is also setup to handle authenticating  users. To learn more about authentication and identity in ASP.NET 5 see :doc:`/security/authentication/index`. For a more complete overview of the structure of an ASP.NET 5 project see :doc:`/conceptual-overview/understanding-aspnet5-apps`. For this tutorial we will focus on adding functionality to our app using MVC and EF.
```
Comment (shirhatti) [inline aspnet/tutorials/your-first-aspnet-application.rst]: For this tutorial -> In this tutorial

File: aspnet/tutorials/your-first-aspnet-application.rst [hunk 2]
```diff
@@ -246,17 +223,15 @@ Run each of the following commands from the **Command Prompt**:
...
 	
-The .NET Version Manager (**dnvm**) is a set of command line utilities that are used to update and configure .NET Runtime. 
-
-`DNX <http://docs.asp.net/en/latest/dnx/overview.html>`_ stands for the .NET Execution Environment. The ``ef`` command is specificed in the *project.json* file of your project. For more information about ``dnvm``, ``dnu``, and ``dnx``, see `DNX Overview <http://docs.asp.net/en/latest/dnx/overview.html>`_.
+Running ``dnu restore`` will restore the package dependencies specified in your **project.json** file. The ``ef`` command is specificed in the *project.json* file of your project. For more information about ``dnvm``, ``dnu``, and ``dnx``, see :doc:`/dnx/overview`.
 
 The "``add Initial``" command creates a migration named "Initial" that adds code to the project, allowing EF to update the database schema. The ``update`` command creates the actual database. After you run this command, the *Migrations* folder of your project will be updated as follows:
```
Comment (shirhatti) [inline aspnet/tutorials/your-first-aspnet-application.rst]: Can you remove the double quotes around the back ticks?

**Comment by rachelappel** [comment]
squirrels away :shipit:

**Comment by shirhatti** [comment]
image

**Pull request merged**